index.html

```diff
@@ -28,6 +26,149 @@ <h2>I'am a Software developer</h2>
   <li><a href=""><i class="fa-solid fa-m"></i></a></li> 
   </ul>
   </section>
-  </main>
+  </section>
+  <section class="recent-work">
+  <div class="top">
+  <h4>My Recent Works</h4>
+  <img src="image/line.png" alt="">
+  </div>
+  <section class="fram-two">
+  <div class="featured-image">
+  <img src="image/Img-Placeholder.png" alt="">
+  </div>
+  <section class="right-block">
+  <h6>Multi-Post Stories</h6>
+  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+  <section class="tags">
+  <img src="image/css.png" alt="">
+  <img src="image/html.png" alt="">
+  <img src="image/bootstrap.png" alt="">
+  <img src="image/ruby.png" alt=""> <br>
+  <button>See Project</button>
+  </section>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
```

Comment:
- [ ] (OPTIONAL) The technologies (HTML, Bootstrap, Ruby) are not buttons so they should just follow the normal design. Also, you should place them in a `ul` list and each of them an `li` for best semantic practices
- [ ] (OPTIONAL) The `See Project` is a button and as a result should have a `button` tag and all the button interactions should be applied to it 👍